### PR TITLE
Homework 1 Comments

### PR DESCRIPTION
**Pull request by madisonm08**
Overall, when redoing the homework after the error, I didn't face any large issues. I decided to copy and paste small sections from my previous homework and commit them separately to try and mimic my workflow from the first time, but it is not exactly the same. 

During this homework, I relied mostly on the guide for markdown linked in the homework instructions to use the syntax (which I called formatting- need to change that), such as heading and bold. I had already played with the software and practiced pushing and pulling projects from R to Git, so that wasn't difficult. I did struggle with embedding the image, but I asked the student next me in lab (forgot to ask his name), and once he told me it needed to be from a url and not my own laptop, I used one from my Instagram successfully. I attempted to find the steps to minimize the size of the photo in R on Google, but was unable to find them. I realized I recognized some of the basic syntax, (headers, emphasis, and block quotes) from Tumblr, where I used to use them frequently.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+# Biography: Madison Martin
+My name is Madison Martin, 
+*but I also go by Maddie as well*.
```
Comment (a-aliseda) [inline README.md:4]: Good use of italics!

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+I will respond to either name, I'm really not that picky. 
+ 
+I am in the MAPSS program and am pursuing the certificate in Computational Social Science.
+My main research interests are stigma, prejudice, and discrimination in academic and workplace contexts.
+Through this class, and over the course of the MAPSS program, I am hoping to:
+
+1. be introduced to the R programming language and gain coding skills
```
Comment (a-aliseda) [inline README.md:11]: Good use of an ordered list

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+My main research interests are stigma, prejudice, and discrimination in academic and workplace contexts.
+Through this class, and over the course of the MAPSS program, I am hoping to:
+
+1. be introduced to the R programming language and gain coding skills 
+
+2. find new tools to improve my efforts in pyschological research, including my master's thesis project 
+
+3. expand my conception and understanding of the role of data in social science 
+
+![Chinese New Year celebration in LA earleir this year](https://scontent-ort2-2.xx.fbcdn.net/v/t1.0-9/52328253_2646248942057291_7498796436759248896_o.jpg?_nc_cat=106&_nc_oc=AQnPPaILyeCG2zs9lAAXRqh9dV3H_9-J6VoGoggEtUMUUt5Ys9o5wc7Z_uaoNf7DO7g&_nc_ht=scontent-ort2-2.xx&oh=71ec929b4db32cbf0d5025dc184a68a5&oe=5E394BD2)
```
Comment (a-aliseda) [inline README.md:17]: The image was embedded correctly

File: README.md [hunk 4]
```diff
@@ -1 +1,20 @@
...
+
+In my spare time, I like to sing, go to concerts (House of Blues in my **favorite** venue), collect pins, and volunteer at animal shelters when I can.
```
Comment (a-aliseda) [inline README.md:19]: Correct use of bold

File: README.md [hunk 5]
```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+# Biography: Madison Martin
```
Comment (a-aliseda) [inline README.md:2]: Uses header

**Review by a-aliseda — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|----------------------|--------------|------------|------------|
| **Coding style**     |          +        |                 |                  |
| **Coding strategy** |       +         |                 |                  |
| **Presentation**       |         +        |                 |                  |
| **Achievement, creativity** |         |      0         |                  |
| **Ease of access**     |                   |       0         |                  |

+ Overall the homework is well written and it contains almost all of what was asked
+ I think two things were missing:
1.  A link to a web page that would have been desirable
2. Your name in the title of the pull request so that you can be identified more easily
+ Congratulations!

File: README.md (hunk 3 above)
Comment (rrodriguezbarron) [reply README.md:17]: Yes, this is the correct way to embed images using a URL

**Review by rrodriguezbarron — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |   x       |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Madison, this is a great submission. 
* Great use of the numbered list
* Your image loaded as it should
* Correct use of bold and italics

File: README.md (hunk 4 above)
Comment (rrodriguezbarron) [inline README.md:19]: House of Blues is really good !

**Pull request closed without merging**